src/rpc/request.h

```diff
@@ -34,19 +34,19 @@ class JSONRPCRequest
     UniValue id;
     std::string strMethod;
     UniValue params;
-    bool fHelp;
+    enum { EXECUTE, GET_HELP, GET_ARGS } mode = EXECUTE;
```

Comment:
I don't think it strictly matters in this case because no other values of this type are ever instantiated outside of this class, but debugging experience has made me prefer named types to anonymous ones.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21035#discussion_r567807599

> I don't think it strictly matters in this case because no other values of this type are ever instantiated outside of this class, but debugging experience has made me prefer named types to anonymous ones.

I slightly prefer not defining an name that's never used for a type that's never referenced, but happy to apply a suggestion if you have a specific one. Maybe `enum Mode { EXECUTE, GET_HELP, GET_ARGS } mode = EXECUTE;`?

Reply:
Sounds good to me.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/21035#discussion_r580900421

> Sounds good to me.

Updated